tsdb/index/inmem/inmem.go

```diff
@@ -14,6 +14,8 @@ package inmem
 import (
 	"errors"
 	"fmt"
+	"github.com/influxdata/influxdb/v2/storage/metrics"
```

Comment:
We don't need to instrument `inmem` index since support for that is being dropped.

Reply:
Yeah, the stub method here is just to satisfy the `Index` interface